ee/clickhouse/models/property.py

```diff
@@ -220,13 +220,17 @@ def filter_element(filters: Dict, prepend: str = "") -> Tuple[List[str], Dict]:
     conditions = []
 
     if filters.get("selector"):
-        selector = Selector(filters["selector"], escape_slashes=False)
-        params["{}selector_regex".format(prepend)] = _create_regex(selector)
-        conditions.append("match(elements_chain, %({}selector_regex)s)".format(prepend))
+        selectors = filters["selector"] if isinstance(filters["selector"], list) else [filters["selector"]]
```

Comment:
Q: When do we need to box this? Do we send non-array values from FE sometimes?

Reply:
When saving actions with a selector for example:

![image](https://user-images.githubusercontent.com/53387/112617998-f75e0400-8e25-11eb-810e-b5adf9268b56.png)

![image](https://user-images.githubusercontent.com/53387/112618062-093fa700-8e26-11eb-9413-c284f3d2cef7.png)

![image](https://user-images.githubusercontent.com/53387/112617961-edd49c00-8e25-11eb-98d2-8a1747396c97.png)